nuxeo-platform-login-openid/src/main/java/org/nuxeo/ecm/platform/oauth2/openid/OpenIDConnectProvider.java

```diff
@@ -177,21 +177,19 @@ public String getAccessToken(HttpServletRequest req, String code) {
             log.error("Error during OAuth2 Authorization", e);
         }
 
-        String type = response.getContentType();
-
-        try {
+        if ("application".equals(response.getMediaType().getType())
```

Comment:
Potential null pointer access: The variable response may be null at this location